src/public/less/pages/documentation.less

```diff
@@ -0,0 +1,337 @@
+@scrn-md: 768px;
```

Comment:
I believe we already have a set of vars defined that can be imported and used.

Reply:
You are right, fixed